packages/compiler-sfc/src/script/context.ts

```diff
@@ -95,6 +96,8 @@ export class ScriptCompileContext {
       scriptSetupLang === 'ts' ||
       scriptSetupLang === 'tsx'
 
+    this.isCE = /\.ce\.vue$/.test(this.descriptor.filename)
```

Comment:
I mean we can add an option to `SFCScriptCompileOptions` instead of asserting it's `xx.ce.vue`.
Like https://github.com/vitejs/vite-plugin-vue/blob/76ffd55cf5a846d60ba3ec4bad3dd7237a0887ae/packages/plugin-vue/src/index.ts#L64

Reply:
I looked at the relevant code of `vite`, and I will eventually use the `createFilter` method of `@rollup/pluginutils` to create a filter, but will there be any problem using `@rollup/pluginutils` directly in the vue compiler? I have no idea. So I simplified the type and related logic of the `customElement` option